CI-CD/production-deployment-with-tests-java/README.md

```diff
@@ -4,8 +4,12 @@
 
 # Vespa Cloud sample applications - Production Deployment with Java Tests
 
-A minimal Vespa Cloud application for deployment into a Production zone -
-with basic Java-tests
+A minimal Vespa Cloud application for deployment into a Production zone - with basic Java-tests
 
-See [Vespa Cloud Automated Deployments](https://github.com/vespa-engine/cloud/blob/master/en/automated-deployments.html)
-for more details.
+See [Vespa Cloud Automated Deployments](https://cloud.vespa.ai/en/automated-deployments) for details.
+
+Test using:
+
+    mvn test -D test.categories=system -D vespa.test.config=ext/test-config.json 
+    mvn test -D test.categories=staging-setup -D vespa.test.config=ext/test-config.json
+    mvn test -D test.categories=staging -D vespa.test.config=ext/test-config.json 
```

Comment:
Since this is for cloud, and in a CI/CD document, I think using the local test config here is more misleading than helpful.

Reply:
Good point! Working on improving the README now